source/val/validation_state.cpp

```diff
@@ -1370,6 +1370,33 @@ bool ValidationState_t::GetPointerTypeInfo(
   return true;
 }
 
+uint32_t ValidationState_t::GetLargestScalarType(uint32_t id) const {
+  const Instruction* inst = FindDef(id);
+  uint32_t size = 0;
+
+  switch (inst->opcode()) {
+    case spv::Op::OpTypePointer:
```

Comment:
I wonder if this is a case that should be rolled into GetBItWIdth

Reply:
Agree, put it in there (along with `OpTypeUntypedPointerKHR`)